pkgs/applications/audio/MBdistortion/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, faust2jack, faust2lv2 }:
+stdenv.mkDerivation rec {
+  name = "MBdistortion-${version}";
+  version = "v1.1";
```

Comment:
Better move the "v" to the rev attribute of `fetchFromGitHub`, as versions should start with a digit.